Microsoft.O365.Security.Native.ETW/Filtering/EventFilter.hpp

```diff
@@ -97,14 +98,20 @@ namespace Microsoft { namespace O365 { namespace Security { namespace ETW {
         }
 
         void EventNotification(const EVENT_RECORD &, const krabs::trace_context &);
+        void ErrorNotification(const EVENT_RECORD&, const std::string&);
 
     internal:
-        delegate void NativeHookDelegate(const EVENT_RECORD &, const krabs::trace_context &);
+        delegate void OnEventNativeHookDelegate(const EVENT_RECORD &, const krabs::trace_context &);
```

Comment:
NIT: This is likely not the correct naming for the delegate. The On is generally reserved for the callback not for the delegate definition or the event definition.

The recommended suffix for the delegate is `EventHandler` but I think it's ok to keep it as Delegate for historical consistency but we should name them
`EventReceivedNativeHookDelegate`
`ErrorReceivedNativeHookDelegate`

See https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-events

Reply:
Good suggestion, updated!

Reply:
Good call, updated!